translate/snippets/detect.go

```diff
@@ -26,13 +24,13 @@ import (
 )
 
 func detectLanguage(text string) (*translate.Detection, error) {
```

Comment:
Optional: add function comments. https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#comment-functions-and-packages

Reply:
Done

Reply:
I think they're all pretty self-explanatory from the parameter names and return types, going to remove the comment